### PR TITLE
Fix README:multiple config incorrect movies path

### PR DESCRIPTION
**Pull request by coolharsh55**

### CODE REVIEW

**Comment by pheyvaer** [comment]
Hi @coolharsh55 

Thanks for PR. Could also add an entry in the changelog? `Fix README:multiple config incorrect movies path` is fine. :smiley:

**Comment by coolharsh55** [comment]
Just add to the current unreleased version notes? (2.0.3)

**Comment by pheyvaer** [comment]
Yes indeed. Something like

```
## Unreleased

### Fixed
- Fix README:multiple config incorrect movies path
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Example config.yaml: cannot read variable writer (see [issue 29](https://github.com/KNowledgeOnWebScale/walder/issues/29))
 - Example config-errors.yaml: 404 view is not used (see [issue 34](https://github.com/KNowledgeOnWebScale/walder/issues/34))
 - Example config-errors.yaml: 404 is given for the wrong reason (see [issue 33](https://github.com/KNowledgeOnWebScale/walder/issues/33))
+- Fix README:multiple config incorrect movies pathFix README:multiple config incorrect movies path
```
Comment (pheyvaer) [inline CHANGELOG.md:15]: It seems that it's added twice. Could you remove the duplicate one? And add a space after `:`? Thanks!

**Pull request merged**